csv_very_large_file.py

```diff
@@ -0,0 +1,149 @@
+"""
+python csv_to_json_very_large_file.py
+
+"""
+# *********************************************************************************************************************
+import click
+import csv
+import json
+
+@click.command()
+@click.option('-f', '--filename', type=str, required=True, help='Input File Path')
+@click.option('-l', '--limit', type=str, required=True, help='Conversation Split Limit')
+@click.option('-u', '--utterance_col', type=str, required=True,
+              help='Column name containing utterances')
+@click.option('-c', '--convo_id_col', type=str, required=False, default='',
+              help='If conversations which is the id otherwise utterances and defaults to hash of utterance_col')
+@click.option('-t', '--created_at_col', type=str, required=False, default='',
+              help='If there is a created date for utterance otherwise defaults to now')
+@click.option('-x', '--unix_date', is_flag=True, type=bool, required=False, default=False,
+              help='If created_at column is in unix epoch format')
+@click.option('-r', '--role_col', type=str, required=False, default='',
+              help='Which column the role in ')
+@click.option('-p', '--role_mapper', type=str, required=False, default='',
+              help='If role column then role mapper in format "source_client:client,source_expert:expert,*:expert"')
+def main(filename: str):
+    file_name = filename.split("/")[-1]
+    file_name_split = file_name.split(".")[0]
+    json_file_name = file_name_split + "-hf-export.json"
+    convo_dict = combine_conversations(file_name)
+    export_to_hf_json_limit(convo_dict,json_file_name,6000)
+
+
+def combine_conversations(filename: str):
```

Comment:
Provide the data type of the values returned. If no values are returned by the function, then provide None

Add docstring to the function explaining what the function does.